frontend/coreproject/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,156 @@
+<script lang="ts">
+	export let anime: any;
+
+	import ScrollArea from '$components/shared/ScrollArea.svelte';
+	import SidebarDetails from '$components/pages/anime_info/SidebarDetails.svelte';
+
+	// icons
+	import PlayCircle from '$icons/PlayCircle.svelte';
+	import Read from '$icons/Read.svelte';
+	import Listen from '$icons/Listen.svelte';
+	import Video from '$icons/Video.svelte';
+	import Download from '$icons/Download.svelte';
+	import Share from '$icons/Share.svelte';
+	import SettingsOutline from '$icons/SettingsOutline.svelte';
+	import Edit from '$icons/Edit.svelte';
+</script>
+
+<div class="grid grid-cols-12 items-start">
+	<div class="col-span-10 grid grid-cols-12 items-end xl:gap-10">
+		<div class="col-span-7 flex items-end gap-10">
+			<img class="xl:w-60 2xl:w-96 xl:rounded-xl 2xl:rounded-3xl" src={anime.cardBackgroundImage} alt={anime.titles.eng} />
+			<div>
+				<h2 class="font-bold 2xl:!text-5xl 3xl:!text-6xl">{anime.titles.eng}</h2>
+
+				<div class="pt-3 3xl:mt-5 flex items-center gap-2 xl:text-xs 2xl:text-sm 3xl:!text-[16px] font-medium text-surface-100">
+					<span>{anime.titles.japanese}</span>
+					<span>-</span>
+					<span>{anime.titles.eng}</span>
+					<span>-</span>
+					<span>{anime.titles.others}</span>
+				</div>
+
+				<div
+					class="pt-4 flex flex-wrap items-center gap-2 pr-5 xl:text-xs 2xl:text-sm 3xl:!text-[16px] 2xl:!leading-none font-semibold"
+				>
+					<span>{anime.type}</span>
+					<span>-</span>
+					<span>{anime.episodes} eps</span>
+					<span>-</span>
+					<span>{anime.status}</span>
+					<span>-</span>
+					<span>{anime.premiered}</span>
+					<span>-</span>
+					<span>{anime.studio}</span>
```

Comment:
Better implementation exists

Reply:
Yea, I will do that like you suggest before
But it need custom styles, are you okay ?

Reply:
Yea, I will do that like you suggest before
But it need custom styles, are you okay ?

Reply:
> Yea, I will do that like you suggest before
> But it need custom styles, are you okay ?

You can also do it with tailwind only